metadata-models/src/main/pegasus/com/linkedin/chart/ChartQuery.pdl

```diff
@@ -0,0 +1,28 @@
+namespace com.linkedin.chart
+
+/**
+ * Information for chart query which is used for getting data of the chart
+ */
+record ChartQuery {
```

Comment:
this can be an aspect of the Chart instead of being referenced in ChartInfo.

Reply:
Thanks, done.